notifications.json

```diff
@@ -0,0 +1,478 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/Messages/{id}": {
```

Comment:
this is in coherence with the SCIM API (which is camelCase with capitalized endpoints). 

Should we chose a different style for the notifications API ?